test_journal.py

```diff
@@ -0,0 +1,144 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import os
+import pytest
+from sqlalchemy import create_engine
+from sqlalchemy.exc import IntegrityError
+
+TEST_DATABASE_URL = os.environ.get(
+    'DATABASE_URL',
+    'postgresql://ndraper2@localhost:5432/test-learning-journal'
+)
+os.environ['DATABASE_URL'] = TEST_DATABASE_URL
+os.environ['TESTING'] = "True"
+
+import journal
```

Comment:
Keep all imports grouped together.  It will save you confusion in the future looking for them.